compiler/rustc_target/src/spec/targets/aarch64_be_unknown_linux_musl.rs

```diff
@@ -0,0 +1,34 @@
+use rustc_abi::Endian;
+
+use crate::spec::{SanitizerSet, StackProbeType, Target, TargetMetadata, TargetOptions, base};
+
+pub(crate) fn target() -> Target {
+    let mut base = base::linux_musl::opts();
+    base.max_atomic_width = Some(128);
+    base.supports_xray = true;
+    base.features = "+v8a".into();
+    base.stack_probes = StackProbeType::Inline;
+    base.supported_sanitizers = SanitizerSet::ADDRESS
+        | SanitizerSet::CFI
+        | SanitizerSet::LEAK
+        | SanitizerSet::MEMORY
+        | SanitizerSet::THREAD;
+
+    Target {
+        llvm_target: "aarch64_be-unknown-linux-musl".into(),
+        metadata: TargetMetadata {
+            description: Some("ARM64 Linux (big-endian) with musl-libc 1.2.5".into()),
+            tier: Some(3),
+            host_tools: Some(false),
+            std: Some(true),
+        },
+        pointer_width: 64,
+        data_layout: "E-m:e-p270:32:32-p271:32:32-p272:64:64-i8:8:32-i16:16:32-i64:64-i128:128-n32:64-S128-Fn32".into(),
+        arch: "aarch64".into(),
+        options: TargetOptions {
+            mcount: "\u{1}_mcount".into(),
+            endian: Endian::Big,
```

Comment:
Is there a reason the `frame_pointer` changes to the aarch64-unknown-linux-musl target wouldn't also apply to this target?

https://github.com/rust-lang/rust/blob/350d0ef0ec0493e6d21cfb265cb8211a0e74d766/compiler/rustc_target/src/spec/targets/aarch64_unknown_linux_musl.rs#L32-L35

Reply:
No substantial reason, I simple hadn't noticed 0c157b51d339fbfe51b6ef21acd4b21452c76f2c yet. I'll integrate both aarch64 changes (non-leaf frame pointers and outline-atomics) into this aarch64_be target